config/environments/development.rb

```diff
@@ -60,4 +62,8 @@
   # Site doesn't have a login, so the CSRF protection isn't helpful
   # In fact, it's problematic because the varying CSRF tokens means you can't get a consistent ETag
   config.action_controller.allow_forgery_protection = false
+
+  config.session_store :cache_store, key: "_sessions_development",
+    compress: true, pool_size: 5, expire_after: 1.month
```

Comment:
You're missing a comma at the end of this line, so the `url` is likely just a hash literal that's not doing anything. Though maybe the `url` line just needs to be deleted. The same thing is in the production file.

```suggestion
    compress: true, pool_size: 5, expire_after: 1.month,
```